orca-igor/src/main/java/com/netflix/spinnaker/orca/igor/tasks/GetBuildPropertiesTask.java

```diff
@@ -62,6 +62,9 @@
             stageDefinition.getMaster(),
             stageDefinition.getJob());
     if (properties.size() == 0) {
+      if (stageDefinition.getMaster().startsWith("travis-")) {
```

Comment:
any chance for `stageDefinition.getMaster()` to be null here?

Reply:
No, it is the name of the buildmaster. It needs to be defined and is fetched from igor.

Reply:
If it was null an exception would have been thrown a few lines up as well.